tests/integration_tests/functional/test_api.py

```diff
@@ -411,12 +411,12 @@ def test_api_machine_config(test_microvm_with_api):
         )
 
     # Test that CPU template errors on ARM.
-    response = test_microvm.machine_cfg.patch(cpu_template="C3")
     if platform.machine() == "x86_64":
+        response = test_microvm.machine_cfg.patch(cpu_template="C3")
         assert test_microvm.api_session.is_status_no_content(response.status_code)
     else:
-        assert test_microvm.api_session.is_status_bad_request(response.status_code)
-        assert "CPU templates are not supported on aarch64" in response.text
+        response = test_microvm.machine_cfg.patch(cpu_template="None")
```

Comment:
What does it actually test? How is it different from not passing the argument?

Reply:
it is. it is not `None` but `"None"`

Reply:
and `"None"` is not `None`

Reply:
I guess it deserves some comment explaining that we check that the "None" template is supported.

Reply:
added comment